ngraph/frontend/paddlepaddle/src/op/softmax.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "softmax.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs softmax(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    auto axis = node.get_attribute<int32_t>("axis");
+                    if (axis < 0)
```

Comment:
I don't see restrictions for negative axis. Doesn't it work with negative axis? https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/SoftMax_1.md

Reply:
The docs says the aixs should be `Range of values: positive integer value` Therefore we do a conversion here.
@luo-cheng2021  Could you provide more info ?

Reply:
Yes, SoftMax spec and implementation all require positive axis.

Reply:
I see. Looks like it could be improved on the opset side (with new version of SoftMax operation)